content/blog/2018/07/2018-07-12-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: "Pipeline as YAML: Alpha release"
+tags:
+- gsoc2018
+- plugin
+- pipeline
+- yaml
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of
+technology, India, Nagpur. I was a member of ACM Chapter and Google student developer club of my
+college. I am passionate about automation.
+
+=== Mentors
+
+* link:https://github.com/oleg-nenashev[Oleg Nenashev] (Org Admin)
+* link:https://github.com/martinda[Martin d'Anjou]
+* link:https://github.com/kwhetstone[Kristin Whetstone]
+* link:https://github.com/grandvizier[Jeff Knurek]
+
+=== Project Summary
+
+This is a GSoC 2018 project.
+
+This project aims to develop a pull request Job Plugin. Users should be able to
+configure job type using YAML file placed in root directory of the
+Git repository being the subject of the pull request. The plugin should interact with various
+platforms like Bitbucket, Github, Gitlab, etc whenever a pull request is created or updated.
+
+Plugin detects the presence of certain types of reports at conventional locations,
+and publish them automatically. If the reports are not present at their respective conventional
+location, the location of the report can be configured in the YAML file.
+
+==== Benefits to the community
+
+* Project administrators will be able to handle pull request builds more easily.
+* Build specifications for pull requests can be written in a concise declarative format.
+* Build reports will be automatically published to Github, Bitbucket, etc.
+* Build status updates will be sent to git servers automatically.
+* Users will not have to deal with pipeline code.
+* If there will be no merge conflicts or build failures, the PR can be merged into target branch.
+
+=== Phase 1 blog post
```

Comment:
It could be shortened as well or just linked from the introduction section

Reply:
done